receiver/k8sclusterreceiver/internal/collection/nodes_test.go

```diff
@@ -47,6 +48,12 @@ func TestNodeMetrics(t *testing.T) {
 
 	testutils.AssertMetrics(t, actualResourceMetrics[0].metrics[1], "k8s.node.condition_memory_pressure",
 		metricspb.MetricDescriptor_GAUGE_INT64, 0)
+
+	testutils.AssertMetrics(t, actualResourceMetrics[0].metrics[2], "k8s.node.allocatable_cpu",
+		metricspb.MetricDescriptor_GAUGE_INT64, 123)
+
+	testutils.AssertMetrics(t, actualResourceMetrics[0].metrics[3], "k8s.node.allocatable_memory",
+		metricspb.MetricDescriptor_GAUGE_INT64, 456)
```

Comment:
Please add `ephemeral-storage` storage as well. I would like to have an example written somewhere with proper snake case

Reply:
Added `ephemeral-storage` in test cases.